File: Adrian/wk01/3-wed/arrays.js
//days of the week

var days_of_the_week = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
console.log("days of the week are: " + days_of_the_week);
days_of_the_week.unshift(days_of_the_week.pop());
console.log("christian days of the week are: " + days_of_the_week);

var weekdays = days_of_the_week.slice(1, 6);
console.log("sober days of the week are: " + weekdays);
var weekends = days_of_the_week.splice(1, 5);
console.log("boozy days of the week are: " + days_of_the_week);

//alternate method would be to shift and pop the weekdays

weekdays.sort();
console.log("sober days in alphabetical order: " + weekdays);

File: Adrian/wk04/1-mon/arrays.rb
days_of_the_week = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"]
puts "1. Create an array of the days of the week"
puts days_of_the_week

day_to_move = days_of_the_week.pop
days_of_the_week.unshift(day_to_move)
puts "2. My calender says the first day is Sunday..."
puts days_of_the_week

weekdays = days_of_the_week[1..5]
weekends = []
weekends.push(days_of_the_week.first).push(days_of_the_week.last)
separated_array = [weekdays, weekends]
puts "3. Create a new array of the days of the week"
puts separated_array

separated_array.shift
puts "4. Remove either the weekdays or the weekends"
puts separated_array

 # no point sorting just weekends
puts "5. Sort the days alphabetically"
puts days_of_the_week.sort